src/centralerekenapplicatie_v1/models.py

```diff
@@ -7,7 +8,7 @@ class CRAMetric(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     message_id = models.TextField()  # Coming from "id" in the message.
     sensor = models.CharField(max_length=255)  # e.g. "CMSA-GAWW-17"
-    timestamp = TimescaleDateTimeField(interval="1 day")  # Timestamp of when the data was recorded in the camera
+    timestamp = TimescaleDateTimeField(interval="1 day", default=timezone.now)  # Timestamp of when the data was recorded in the camera
```

Comment:
Waarom heb je hier een default toegevoegd? Dat is niet nodig en kan leiden tot geschiedvervalsing.